modules/ROOT/pages/as2/as2-connector.adoc

```diff
@@ -0,0 +1,133 @@
+= AS2 Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: Premium
+
+The Mule Runtime supports the AS2 protocol through the AS2 Connector. The AS2 connector enables trading partners transmit and receive AS2 messages with EDI-X12, EDIFACT, XML or binary payloads.
```

Comment:
Either Mule or Mule runtime engine (no Mule Runtime)

Reply:
Fixed by you.